std/v2/range/primitives.d

```diff
@@ -0,0 +1,33 @@
+module std.v2.range.primitives;
+
+/**
+@@@TODO@@@ Publicly imported names across modules should copy or link those
```

Comment:
http://phobos.dpldocs.info/std.string.html#public-imports

Reply:
@adamdruppe that's close. How easy would it be to modify dpldocs to:

- actually link to the individual names imported?
- maybe copy the entire documentation?
- maybe make the documentation available as a macro for subsequent use?

Reply:
All pretty easy, I just never got around to it (I personally kinda hate selective imports. They're .... really weird when you get into them. I prefer static imports. But since I don't personally use it and there's been no user demand this just never became enough of a priority to actually do. But it'd prolly only take an hour or two.)